src/bio/README.md

```diff
@@ -25,20 +41,165 @@ NVMe SSDs are assigned to each DAOS server xstream. SPDK blobstores are created
 * Association of SPDK I/O channels with DAOS server xstreams:
 Once SPDK I/O channels are properly associated to the corresponding device, NVMe hardware completion pollers are integrated into server polling ULTs.
 
-  ## Per-Server Metadata Management (SMD)
-  One of the major subcomponenets of the BIO module is per-server metadata management. The SMD submodule consists of a PMDK pmemobj pool stored on SCM used to track each DAOS server's local metadata.
+<a id="3"></a>
+## Per-Server Metadata Management (SMD)
+   One of the major subcomponenets of the BIO module is per-server metadata management. The SMD submodule consists of a PMDK pmemobj pool stored on SCM used to track each DAOS server's local metadata.
 
-  Currently, the persistent metadata tables tracked are :
-  - **NVMe Stream Table**: NVMe SSD to DAOS server xstream mapping (local PCIe attached NVMe SSDs are assigned to different server xstreams to avoid hardware contention)
+   Currently, the persistent metadata tables tracked are :
+  - **NVMe Device Table**: NVMe SSD to DAOS server xstream mapping (local PCIe attached NVMe SSDs are assigned to different server xstreams to avoid hardware contention)
   - **NVMe Pool Table**: NVMe SSD, DAOS server xstream, and SPDK blob ID mapping (SPDK blob to VOS pool:xstream mapping)
-  - **NVMe Device Table**: (in progress) NVMe SSD to device status mapping
-  On DAOS server start, these tables are loaded from persistent memory and used to initialize new, and load any previous blobstores and blobs. Alos, there is potential to expand this module to support other non-NVMe related metadata in the future.
+  - **NVMe Device State Table**: NVMe SSD to device state mapping (supported states are NORMAL or FAULTY)
+   On DAOS server start, these tables are loaded from persistent memory and used to initialize new, and load any previous blobstores and blobs. Also, there is potential to expand this module to support other non-NVMe related metadata in the future.
 
-  ## DMA Buffer Management
+   Useful Admin Commands to Query SMD:
+   <a href="#31">dmg storage query smd</a> [used to query both SMD device table and pool table]
+
+<a id="4"></a>
+## DMA Buffer Management
   BIO internally manages a per-xstream DMA safe buffer for SPDK DMA transfer over NVMe SSDs. The buffer is allocated using the SPDK memory allocation API and can dynamically grow on demand. This buffer also acts as an intermediate buffer for RDMA over NVMe SSDs, meaning on DAOS bulk update, client data will be RDMA transferred to this buffer first, then the SPDK blob I/O interface will be called to start local DMA transfer from the buffer directly to NVMe SSD. On DAOS bulk fetch, data present on the NVMe SSD will be DMA transferred to this buffer first, and then RDMA transferred to the client.
 
+<a id="5"></a>
 ## NVMe Threading Model
-![/doc/graph/NVME_Threading_Model.PNG](/doc/graph/NVME_Threading_Model.PNG "NVMe Threading Model")
```

Comment:
Is the old PNG file removed from repository?

Reply:
thanks for catching this. I will update patch to remove the old file.